### ISSUE TITLE
Add stubs to redirect from old website pages

### ISSUE THREAD

**Issue by jlstevens** [ioam/holoviews#1750]
People are finding webpages that were part of the old website but no longer exist on the new website. We should copy those old pages off S3 to have a record of them and replace them with stubs to redirect people to the new website.

**Comment by jordansamuels** [comment]
I suggest a few options for redirects.  


### Option A - simple redirect to the new home page
 A uniform redirect from all former pages to the new home page, such as the following (or equivalent `.rst`):
```html
The HoloViews documentation has moved to the 
<a href="http://holoviews.org">HoloViews home page</a>. 
```

### Option B - redirect to the new page with some guidance on new structure
A uniform redirect from all former pages to the new home page, with more guidance on the new structure (probably more useful for previous users).  Much wording stolen from new home page.

```html
The HoloViews documentation has moved to the 
<a href="http://holoviews.org">HoloViews home page</a>.  <br>

The <a href="/gallery/">Gallery</a> has demos and apps, and the 
<a href="/reference/">Reference Gallery</a> has entries for all HoloViews components.  
The  <a href="/getting_started/">Getting-Started</a> guide explains basic concepts and 
how to start using HoloViews, and the <a href="/user_guide/">User Guide</a> goes more
deeply into key concepts form HoloViews.
```

### Option C - customized redirects based on context
This would be more work and involve deciding what the most relevant new content is.  For example, http://holoviews.org/Tutorials/Dynamic_Map.html could contain something like this:

```html
The HoloViews documentation has moved to the 
<a href="http://holoviews.org">HoloViews home page</a>.  <br>

For info on DynamicMap, see 
<a href="/reference/containers/bokeh/DynamicMap.html">its reference page</a>, 
as well as its usage in <a href="/getting_started/Live_Data.html">Live Data</a>,
<a href="user_guide/Responding_to_Events.html>Responding to Events</a>,
<a href="/user_guide/Data_Pipelines.html">Data Pipelines</a>, and
<a href="/user_guide/Dashboards.html"></a>Creating interactive dashboards</a>
```
I chose those to link by doing `git grep -c DynamicMap -- 'examples/**/*.ipynb` in the current code, and browsing a bit.  

This is more work, however (a) I'd be willing to do it, and (b) it might be useful to also put these links into the Reference section on the new site.  That way, if someone wants to see more doc context on DynamicMap, they can easily browse to it.

**Comment by jlstevens** [comment]
@jordansamuels Thanks for outlining the options.

I think I am in favor of getting Option A initially for a quick fix at which point we can spend more time to do Option B or C properly. The tutorials page is the one place I know needs this but there may be others, I don't know of.

```
The HoloViews documentation has moved to the 
<a href="http://holoviews.org">HoloViews home page</a>. 
```

Something like that is fine but maybe it should say something about the current page being part of the old docs before pointing to the new docs.

**Comment by jordansamuels** [comment]
@jlstevens good point about indicating that the current page is old.  I was thinking that some sort of warning style like the blue background we saw in e.g. the [Bokeh Backend tutorial](https://github.com/ioam/holoviews/blob/master/doc/Tutorials/Bokeh_Backend.ipynb)

```html
<div class="alert alert-info" role="alert">
    This page is no longer part of the HoloViews documentation, which has moved to the 
<a href="http://holoviews.org">HoloViews home page</a>. 
</div>
```

which might render as (pasting screenshot since GitHub markup evidently doesn't allow style/color:

![image](https://user-images.githubusercontent.com/610909/28581045-d5c0b0ec-7126-11e7-8af3-175cb3dead1c.png)

**Comment by jlstevens** [comment]
Another good suggestion. Maybe:

'This page is no longer part of the HoloViews documentation, which has moved to the updated [holoviews.org] website'

**Comment by michaelaye** [comment]
Are all the dead links on this page another symptom of this issue?

http://holoviews.org/Tutorials/index.html

**Comment by jordansamuels** [comment]
I got this email and I don't see the comment here, so I'll post it here.  It's a good question!

> ---------- Forwarded message ----------

**Comment by michaelaye** [comment]
I deleted my comment, because there's another GH issue that focuses on the question on broken links already.

**Comment by jlstevens** [comment]
I've added a redirect page to [Tutorials](http://holoviews.org/Tutorials/) and I've also added a [robots.txt](http://holoviews.org/robots.txt) to try to delist that page from google. There may be other parts of the website that need this treatment but Tutorials is the only section I currently know about that needs this.

**Comment by jbednar** [comment]
Would it not be better just to make Tutorials redirect to the User Guide?

**Comment by jlstevens** [comment]
Maybe we should have links for both? What message would you propose?

**Comment by jbednar** [comment]
Not sure what you mean; I meant an HTTP 301 permanent redirect; does that need a message?

**Comment by philippjfr** [comment]
The robots.txt seems to have worked and ioam.github.io/holoviews no longer seems to be indexed by google.
